src/views/bootsources/utils/utils.ts

```diff
@@ -0,0 +1,46 @@
+import useInstanceTypesAndPreferences from '@catalog/CreateVMHorizontalNav/components/CreateFromInstanceTypes/hooks/useInstanceTypesAndPreferences';
+import {
+  DEFAULT_INSTANCETYPE_LABEL,
+  DEFAULT_PREFERENCE_LABEL,
+} from '@catalog/CreateVMHorizontalNav/components/CreateFromInstanceTypes/utils/constants';
+import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { V1alpha2VirtualMachineClusterPreference } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { DataSourceModelGroupVersionKind } from '@kubevirt-utils/models';
+import { ANNOTATIONS } from '@kubevirt-utils/resources/template';
+import {
+  K8sResourceCommon,
+  Operator,
+  useK8sWatchResource,
+  WatchK8sResult,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+export const useBootableVolumes = (): WatchK8sResult<V1beta1DataSource[]> =>
```

Comment:
I thought it should be fetched only from `openshift-virtualization-os-images` namespace? 
nit suggestion: I would move all hooks from here to bootsources/hooks directory
I would also rename `bootsources` directory to `bootablevolumes` to align with the name

Reply:
> I thought it should be fetched only from openshift-virtualization-os-images namespace?

If you mean adding specifying the namespace when fetching the resource, then yes, I should!

> nit suggestion: I would move all hooks from here to bootsources/hooks directory
> I would also rename bootsources directory to bootablevolumes to align with the name

Of course. This is just because they've called it "Boot sources" and not "Bootable volumes" like it is now, so everything I've implemented, was aligned with that. I just forgot to rename the `bootsources` dir, after the change. Good catch!

Reply:
> If you mean adding specifying the namespace when fetching the resource, then yes, I should! (it works without that anyway winkinnocent)

you're right, as the admin user, it really should work but non-admin users can reach only DS in that specific namespace :)